### PR TITLE
PWX-21086, PWX-21087: Initial License SKU/Feature test

### PR DESCRIPTION
**Pull request by pault84**
Signed-off-by: Paul Theunis <paul@portworx.com>

**What this PR does / why we need it**:
PWX-21086: 
Checking the activated license against PX-Essentials FA/FB

PWX-21087: 
Range over the returned licensed features and compare them with our hardcoded expectations

PWX-21091
Add Torpedo tests that deletes the FA/FB secret

PWX-21095
Add Torpedo test that verifies we do not fallback to PX-Essentials

**Which issue(s) this PR fixes** (optional)
Closes #
PWX-21086
PWX-21087
PWX-21095
PWX-21091

**Passed Jenkins Jobs:**

1. _DeleteSecretLicExpiryAndRenewal_ http://jenkins.pwx.dev.purestorage.com/view/Control%20Plane/job/Torpedo/job/2-tp-nextpx-pure-esse/12/
2. _DeleteSecretRebootAllNodes_ http://jenkins.pwx.dev.purestorage.com/view/Control%20Plane/job/Torpedo/job/2-tp-nextpx-pure-esse/14/

### CODE REVIEW

File: drivers/volume/portworx/portworx.go [hunk 1]
```diff
@@ -2335,6 +2354,29 @@ func (d *portworx) EstimateVolumeExpand(apRule apapi.AutopilotRule, initialSize,
...
+	lic, err := licenseMgr.Status(d.getContext(), &pxapi.PxLicenseStatusRequest{})
+	if err != nil {
+		return licenseSummary, err
+	}
+
+	licenseSummary.SKU = lic.GetStatus().GetSku()
+
+	features, err := featureMgr.Enumerate(d.getContext(), &pxapi.PxLicensedFeatureEnumerateRequest{})
+	if err != nil {
+		logrus.Infof("XXX: %v - %v", features, err)
```
Comment (vinayakshnd) [inline drivers/volume/portworx/portworx.go:2372]: Should it be `logrus.Errorf()` ?

Comment (pault84) [reply drivers/volume/portworx/portworx.go:2372]: that should actually be removed my bad

**Comment by pault84** [comment]
Going to remove the limit test from this PR so we can close some tickets.

**Review by thiguetta — APPROVED** [review]
LGTM

**Pull request merged**